lighthouse-core/test/runner-test.js

```diff
@@ -295,4 +296,96 @@ describe('Runner', () => {
       assert.strictEqual(AuditClass.meta.name, auditExpectedName);
     });
   });
+
+  it('results include artifacts when given artifacts and audits', () => {
+    const url = 'https://example.com';
+    const config = new Config({
+      audits: [
+        'is-on-https'
+      ],
+
+      artifacts: {
+        HTTPS: {
+          value: true
+        }
+      }
+    });
+
+    return Runner.run({}, {url, config}).then(results => {
+      if (!results.artifacts instanceof Object) {
+        assert.ok(false);
+        return;
+      }
+
+      assert.strictEqual(results.artifacts.HTTPS.value, true);
+
+      for (const method in computedArtifacts) {
+        if (!computedArtifacts.hasOwnProperty(method)) {
```

Comment:
could you just do a loop over `Object.keys(computedArtifacts)` here to avoid `hasOwnProperty`, etc?